.github/workflows/publish.yml

```diff
@@ -189,17 +189,17 @@ jobs:
       fail-fast: false
       matrix:
         arch: [arm64, amd64]
-        hv: [kvm, xen]
+        hv: [kvm, xen, kubevirt]
```

Comment:
At this phase we are only building eve image and publish_sources, the packages should be built and publish already by the previous jobs, so you need to add kubevirt to the matrix as well (see lines 99 to 107 of this file)....

Reply:
@andrewd-zededa , you still didn't fix it here... you need to first build pkgs (`make pkgs`) for kubevirt (see from line 28 of this file), at this part we are just building eve image (`make eve`). Also, I understood that you are going to build kubevirt only for amd64 for now.... not for arm64....